elements/rh-band/src/rh-band.html

```diff
@@ -0,0 +1,15 @@
+<div class="rh-band__wrapper">
+  <div class="rh-band__container">
+    ${this.has_slot("aside") && this.asidePosition.includes("full") && this.asidePosition.includes("top") ? `<slot class="rh-band__aside" name="aside"></slot>` : ''}
+    <section class="rh-band__main">
+      ${this.has_slot("header") && !(this.has_slot("header_title") || this.has_slot("header_heading") || this.has_slot("header_summary")) ? `<slot class="rh-band__header" name="header"></slot>` : ''}
```

Comment:
@castastrophe hmm,  I'm not seeing slots for header_title, header_heading or header_summary , can you write me a simplified version of what this line of logic does?

Reply:
I migrated to the [typography="title"] syntax.  Documentation and clean-up has not happened yet but if you look in storybook you'll see the fields and markup I'm proposing.